pkgs/top-level/python-packages.nix

```diff
@@ -2524,6 +2524,8 @@ in {
     doCheck = false; # lazy packager
   };
 
+  csscompressor = callPackage ../development/python-modules/csscompressor.nix;
```

Comment:
I expect this should look more like `callPackage foo { }`, as it is this is a partially applied function.